copy/copy.go

```diff
@@ -94,8 +94,15 @@ type Options struct {
 	DestinationCtx   *types.SystemContext
 }
 
-// Image copies image from srcRef to destRef, using policyContext to validate source image admissibility.
-func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageReference, options *Options) error {
+// Image copies image from srcRef to destRef, using policyContext to validate
+// source image admissibility.
+func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageReference, options *Options) (retErr error) {
+	// XXX this function uses an output parameter for the error return value.
+	// Setting this and returning is the ideal way to return an error.
```

Comment:
Looking at https://golang.org/ref/spec#Return_statements , even with a named return value it would be perfectly fine to just `return somerrorvalue` AFAICS.

(Also, doesn’t `XXX` mean “this is bad”? Perhaps just `NOTE:` or something like that.)